src/examples/button/button.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"machine"
+	"runtime"
+)
+
+func main() {
+	led := machine.GPIO{machine.LED}
+	led.Configure(machine.GPIOConfig{Mode: machine.GPIO_OUTPUT})
+
+	button := machine.GPIO{8}
+	button.Configure(machine.GPIOConfig{Mode: machine.GPIO_INPUT})
+
+	led.High()
+
+	for {
+		if button.Get() {
+			led.Low()
+		} else {
+			led.High()
+		}
```

Comment:
This could be simplified to:
```go
led.Set(!button.Get())
```
but maybe the existing code is more obvious (and optimized to the same thing anyway, I tested it) so I leave it up to you.

Reply:
Change made, thanks for the feedback.